R/pipe_continuation_linter.R

```diff
@@ -0,0 +1,41 @@
+#' @describeIn linters check that all commas are followed by spaces, but do not
+#' have spaces before them.
+#' @importFrom xml2 xml_find_all as_list
+#' @export
+pipe_continuation_linter <- function(source_file) {
```

Comment:
This text is actually wrong, it is residual from the commas linter. This linter does nothing with spaces, just determines if a pipe is either all on one line or spread across multiple lines with one action per line.